content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/_cq_dialog/.content.xml

```diff
@@ -72,6 +72,15 @@
                                         jcr:primaryType="nt:unstructured"
                                         sling:resourceType="granite/ui/components/coral/foundation/container">
                                         <items jcr:primaryType="nt:unstructured">
+                                            <label
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                                fieldDescription="Value of aria-label attribute, which describes purpose of the set of tabs."
+                                                fieldLabel="Tabs Label"
+                                                name="./label"
+                                                value=""
+                                                emptyText="Enter value of label."
```

Comment:
Same comment as with the Navigation, would remove this or provide a good default example.

Reply:
Solved by removing